faq.md

```diff
@@ -93,8 +93,8 @@ The original MHC implementation assumed that using the Machine API to destroy
 the bad node and replace it with a new one was the only necessary recovery
 mechanism.
 
-The medik8s team partnered with Ericsson to convince the sig-cluster community
-that other mechanisms were needed (particularly on bare metal) and together we
+The medik8s team has partnered with Ericsson to convince the sig-cluster community
+that other mechanisms have been needed (particularly on bare metal), and together we have
```

Comment:
I would not make any of these changes

Reply:
The tense changes might be discarded, but don't you think there is a need for a comma (`,`). It is a very long sentence IMO.

Reply:
The tense changes might be discarded, but don't you think there is a need for a comma (`,`) ? It is a very long sentence IMO.

Reply:
```suggestion
The medik8s team has partnered with Ericsson to convince the sig-cluster community
that other mechanisms have been needed (particularly on bare metal). Together we have
```